tests/parser/functions/test_create_functions.py

```diff
@@ -358,9 +357,11 @@ def should_fail(target: address, arg1: String[129], arg2: Bar):
     assert FooContract(d.created_address()).bar() == BAR
 
     # Foo constructor should fail
-    FOO = b"\x01" * 129
+    FOO = "01" * 129
     BAR = ("",)
-    assert_tx_failed(lambda: d.should_fail(f.address, FOO, BAR))
+    sig = keccak("should_fail(address,string,(string))".encode()).hex()[:10]
+    encoded = abi_encode("(address,string,(string))", [f.address, FOO, BAR]).hex()
```

Comment:
same. i think it might be good to drop the `abi_encode` helper in conftest anyways?

Reply:
> same.

Do you mean that `FOO` and `BAR` can be inlined?

> i think it might be good to drop the abi_encode helper in conftest anyways?

I think the `abi_encode` helper may be useful in saving us from having to make the same import in multiple files (5 at the moment). Also, we can easily switch to another library by changing this helper only. What do you think?

Reply:
> Do you mean that FOO and BAR can be inlined?

`(f.address, FOO, BAR)`

> I think the abi_encode helper may be useful in saving us from having to make the same import in multiple files (5 at the moment). Also, we can easily switch to another library by changing this helper only. What do you think?

right but since the way you changed `abi_encode` changed only the call but not how the arguments are passed, all the usages of `abi_encode` need to update their parameters anyways. unless i am missing something?

Reply:
> all the usages of abi_encode need to update their parameters anyways

There shouldn't be a need to update the parameters. I noticed that I left out `tests/parser/features/test_logging.py` and `tests/parser/features/test_reverting.py` which were using `eth_abi` and its new format, and have just changed them in https://github.com/vyperlang/vyper/pull/3326/commits/eb223a22711e627ee520c74deb3efd9d260063f1. Could you have been referring to those? Otherwise, the places where `abi_encode` helper is used should be unchanged like [here](https://github.com/vyperlang/vyper/blob/20a14785089c4eeb73015bc52049edfe97f5e998/tests/parser/functions/test_abi_decode.py#L68).